Orgs locs docs (#3281)
theforeman/foreman-documentation: guides/common/modules/con_organizations-and-locations-in-project.adoc

[id="Organizations-and-Locations-in-{ProjectNameID}_{context}"]
= Organizations and locations in {ProjectName}

On your {ProjectServer}, you can define _organizations_ and _locations_ to help organize content, hosts, and configurations.
Organizations and locations enable you to arrange {Project} resources into logically structured groups.
For example, you can create groups based on ownership, purpose, content, or security level.
ifdef::satellite[]

> You can create and manage multiple organizations through {ProjectName}, then divide and assign your Red Hat subscriptions to each individual organization.

Reviewer: ```suggestion
You can create and manage multiple organizations through {ProjectName}, then divide and assign your Red{nbsp}Hat subscriptions to each individual organization.
```

+
By creating locations, you can define geographical sites where hosts are located.
For example, this is useful in environments with multiple data centers.
+
You can use locations to map the network infrastructure to prevent incorrect host placement or configuration.
While you cannot assign a subnet, domain, or compute resources directly to a {SmartProxyServer}, you can assign them to a location.
+
Unlike organizations, locations can have a hierarchical structure.

{ProjectServer} defines all locations and organizations.

> Each respective {Project} {SmartProxyServer} synchronizes content and handles configuration of systems in a different location.

Reviewer: ```suggestion
Each {SmartProxyServer} synchronizes content and handles configuration of hosts in a different location.
```

> The main {ProjectServer} retains the management function, while the content and configuration is synchronized between the main {ProjectServer} and a {Project} {SmartProxyServer} assigned to certain locations.

Reviewer: ```suggestion
Your {ProjectServer} retains the management function, while the content and configuration is synchronized between your {ProjectServer} and {SmartProxyServers} assigned to certain locations.
```

.Example of using organizations and locations in {Project}
====

> The structure of a multi-national company includes the Finance, Marketing, and Sales departments. The company operates across United States, United Kingdom, and Japan.

Reviewer: ```suggestion
The structure of a multi-national company includes the Finance, Marketing, and Sales departments.
The company operates across United States, United Kingdom, and Japan.
```

> The system administrator creates the following _organizations_ on their {ProjectServer}:

Reviewer: I am unsure if emphasis is needed. Your call.

Author: Dropped.
